src/EnergyPlus/ConfiguredFunctions.hh

```diff
@@ -49,12 +49,13 @@
 #define CONFIGURED_FUNCTIONS_HH
 
 #include <string>
+#include <EnergyPlus/FileSystem.hh>
 
 namespace EnergyPlus {
 
-std::string configured_source_directory();
+fs::path configured_source_directory();
```

Comment:
Yes, they should.  That was added in a bit of a rush, and ended up the way they are because they needed to be fixed in a hurry.  If you want to clean it up that is totally fine.

Reply:
They can just be `constexpr char *` variables now actually, which is even simpler.

Reply:
If I make them `constexpr char *` then all calls would have to look like `fs:path(configured_source_directory()) / xxx`

Reply:
`constexpr string_view` would be better than `constexpr char *` but in either case, yes you'd have to call your conversion function to use them.